scriptmodules/ports/steamlink.sh

```diff
@@ -13,11 +13,11 @@ rp_module_id="steamlink"
 rp_module_desc="Steam Link for Raspberry Pi 3 or later"
 rp_module_licence="PROP https://steamcommunity.com/app/353380/discussions/0/1743353164093954254/"
 rp_module_section="exp"
-rp_module_flags="!all rpi3 rpi4"
+rp_module_flags="!all rpi3 rpi4 32bit"
```

Comment:
This will disable for all platforms, then enable for rpi3 and rpi4 and then any 32bit platforms. I don't think that's what you want as it would mean the module is enabled for 64bit on the rpi3/rpi4.

`!all rpi3 rpi4 !64bit` would enable for rpi3 and rpi4 but disable for 64bit.

Reply:
You know, `!64bit` was my first choice. I should have known better. Modified.